res/sections/65-Lez18042018.tex

```diff
@@ -123,32 +148,40 @@ \section{Suggerimenti vari}
 \subsection{Mostrare energia}
 
 Essere attivi è il primo modo per mantenere il pubblico attivo. È importante
-rompere il ritmo, variando velocità e tono di vocie. Sul muoversi nel palco ci
-sono diverse metodologie e scuole di pensiero. In definitiva, attirare
-l'attenzione è in modo per farsi ricordare.
+rompere il ritmo, variando velocità e tono di voce, si può attirare
+l'attenzione sia alzando il volume della voce che abbassandolo. Sul muoversi
```

Comment:
Questa frase non sta tantissimo in piedi :sweat_smile: 
Io la riscriverei così: `È importante rompere il ritmo: variando la velocità e il tono di voce è infatti possibilie attirare l'attenzione degli ascoltatori. Riguardo al muoversi [...]`